cypress/integration/examples/misc.spec.js

```diff
@@ -89,8 +89,11 @@ context('Misc', () => {
         scale: false,
         disableTimersAndAnimations: true,
         screenshotOnRunFailure: true,
-        beforeScreenshot () { },
-        afterScreenshot () { },
+        // TODO: remove this when Cypress typedefs are fixed
+        // https://github.com/cypress-io/cypress/pull/7445
+        // @ts-ignore
+        onBeforeScreenshot () { },
+        onAfterScreenshot () { },
```

Comment:
hmm, this is weird - it passes TS check with a single ts-ignore, not above each line?

Reply:
it's strange, one ts-ignore seems to take care of both